docs-requirements.txt

```diff
@@ -25,8 +25,8 @@ wrapt>=1.0.0,<2.0.0
 celery>=4.0
 psutil~=5.7.0
 boto~=2.0
-google-cloud-trace >=0.23.0
-google-cloud-monitoring>=0.36.0
 botocore~=1.0
 starlette~=0.13
-fastapi~=0.58.1
\ No newline at end of file
+fastapi~=0.58.1
+opentelemetry-exporter-cloud-trace==0.10b0
+opentelemetry-exporter-cloud-monitoring==0.10b0
```

Comment:
If this is just for the docs to build, can we add a follow up issue to remove?

Reply:
Yeah it is. The community repo will always have this dependancy as long as we want Cloud Trace to be present in the community docs (which we should).

Reply:
I guess it depends how much documentation we leave in there. If we just put a link it shouldn't be needed

Reply:
Yeah we can change it later. I definitely don't want any logic changes in this PR! It seems you already made an issue, so I'll link it here for reference https://github.com/GoogleCloudPlatform/opentelemetry-operations-python/issues/27